libsolidity/formal/SMTEncoder.cpp

```diff
@@ -2442,6 +2442,42 @@ SourceUnit const* SMTEncoder::sourceUnitContaining(Scopable const& _scopable)
 	solAssert(false, "");
 }
 
+vector<ASTPointer<Expression const>> SMTEncoder::sortNamedArguments(FunctionCall const& _funCall)
+{
```

Comment:
Copied from `codegen/ExpressionCompiler`

Reply:
Maybe it makes sense to make it a member of `FunctionCall`, let's see what reviewer says

Reply:
Is this supported in IRCodegen?

Reply:
> Copied from codegen/ExpressionCompiler

Do you mean https://github.com/ethereum/solidity/blob/develop/libsolidity/codegen/ExpressionCompiler.cpp#L552 ?

Reply:
Yes that's what I mean

Reply:
> Is this supported in IRCodegen?

It does and has the same sorting: https://github.com/ethereum/solidity/blob/develop/libsolidity/codegen/ir/IRGeneratorForStatements.cpp#L793

Since this would be now duplicated in three places, I think a helper could be better.

Reply:
Agree, but where should it say though? AST or somewhere in codegen?

Reply:
@chriseth any opinion where this helper should go?

Reply:
ast annotations? ast? analysis directory?